docs/hellonode.md

```diff
@@ -133,74 +133,114 @@ A kubernetes **pod** is a group of containers, tied together for the purposes of
 Create a pod with the `kubectl run` command:
 
 ```shell
-kubectl run hello-node \
+$ kubectl run hello-node \
     --image=gcr.io/PROJECT_ID/hello-node:v1 \
     --port=8080
-CONTROLLER  CONTAINER(S)  IMAGE(S)                     SELECTOR        REPLICAS
-hello-node  hello-node    gcr.io/..../hello-node:v1    run=hello-node  1
+deployment "hello-node" created
 ```
 
-Now is probably a good time to run through some of the following interesting kubectl commands (none of these will change the state of the cluster, full documentation is available [here](https://cloud.google.com/container-engine/docs/kubectl/)):
+As shown in the output, the `kubectl run` created a **deployment** object.  Deployments are the recommended way for managing creation and scaling of pods.  In this example, a new deployment manages a single pod replica running the *hello-node:v1* image.
+
+To view the deployment we just created:
+
+```shell
+$ kubectl get deployments
+NAME         DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
+hello-node   1         1         1            1           3m
+```
+
+To view the pod created by the deployment:
 
 ```shell
 $ kubectl get pods
-$ kubectl logs
-$ kubectl cluster-info
-$ kubectl config view
+NAME                         READY     STATUS    RESTARTS   AGE
+hello-node-714049816-ztzrb   1/1       Running   0          6m
+```
+
+To view the stdout / stderr from a pod (hello-node image has no output, so logs will be empty in this case):
+
+```shell
+$ kubectl logs <POD-NAME>
+```
+
+To view metadata about the cluster:
+
+```shell
+$ kubectl get cluster-info
```

Comment:
The link to https://cloud.google.com/container-engine/docs/kubectl/ (which redirects to http://kubernetes.io/docs/user-guide/kubectl-overview/) is gone too.